src/components/EnhancedChatInterface.tsx

```
@@ -60,9 +60,29 @@
 import { githubService } from '@/lib/github-service';
 import DiagramMessageComponent from './DiagramMessageComponent';
 
+// Performance utility functions
+const throttle = <T extends (...args: any[]) => any>(func: T, limit: number): T => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)